traffic/traffic_info.hpp

```diff
@@ -78,4 +80,22 @@ class TrafficInfo
   Coloring m_coloring;
   MwmSet::MwmId m_mwmId;
 };
+
+class RoutingObserver
```

Comment:
I cannot. TrafficManager is map subsystem. I'd like routing does not depends on map. What can I do:
- leaf it in traffic_info.hpp (and in traffic namespace)
- rename it Observer (traffic::Observer)